clang/include/clang/Sema/Sema.h

```diff
@@ -358,7 +359,9 @@ class SYCLIntegrationHeader {
     int Info = 0;
     // Offset of the captured parameter value in the lambda or function object.
     unsigned Offset = 0;
-
+    // NumOpenCLParams to capture number of parameters in Accessor/Sampler init
+    // method.
+    unsigned NumOpenCLParams = 1;
```

Comment:
If we remove the default constructor below, we can make initialization of this simpler, so remove line 365.